frontend/src/components/Form.js

```diff
@@ -41,11 +41,41 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Validates from inputs on submission and alerts user on error
+ * @return {Boolean} true or false based on validity of inputs
+ * @param {String} role
+ * @param {String} productArea
+ * @param {Number} duration
+ * @param {Number} timeAvailableUntilMilliseconds
```

Comment:
change to endTimeAvailable eventually?

Reply:
Yes, I changed this in the following PR #64 . Thanks!